libs/components/src/lib/alert/alert.scss

```
@@ -60,7 +60,7 @@ $alert-transition-delay: --transition-delay;
 		transform: scale(1);
 	}
 
-	min-inline-size: var(#{variables.$alert-min-inline-size}, 420px);
+	min-inline-size: var(#{variables.$alert-inline-size}, 420px);
```

```suggestion
	min-inline-size: var(#{variables.$alert-min-inline-size}, 420px);
	max-inline-size: var(#{variables.$alert-max-inline-size}, fit-content);
```
if no max-width is set if there's lots of text the alert can be beryyyy wide :)

I'm being lazy adding this to all the places that are needed - variable + readme

---

good point. I'll add that.